src/test/java/com/answerdigital/benhession/academy/answerkingweek2/controllers/CategoryControllerTest.java

```diff
@@ -0,0 +1,113 @@
+package com.answerdigital.benhession.academy.answerkingweek2.controllers;
+
+import com.answerdigital.benhession.academy.answerkingweek2.request.AddCategoryRequest;
+import com.answerdigital.benhession.academy.answerkingweek2.request.UpdateCategoryRequest;
+import com.answerdigital.benhession.academy.answerkingweek2.services.CategoryService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+@ContextConfiguration(classes = {CategoryController.class})
+@ExtendWith(SpringExtension.class)
+class CategoryControllerTest {
+    @Autowired
```

Comment:
I think if we just inject @Autowired
    MockMvc mockMvc; 
then we don't need 
@Autowired
 private CategoryController categoryController;
nor @ContextConfiguration(classes = {CategoryController.class})
Instead of @ContextConfiguration(classes = {CategoryController.class}) we can put @WebMvcTest(CategoryController.class)